docs/Reference/API-Methods.md

```diff
@@ -5001,6 +4999,129 @@ specified group of sender and recipients.
       "result": "0x1"
     }
     ```
+### priv_getFilterChanges
+
+Polls the specified filter for a private contract and returns an array of changes that have occurred 
+since the last poll.
+
+Privacy groups do not have blocks and private transactions cannot be pending so unlike 
+[`eth_getFilterChanges`](#eth_getfilterlogs), `priv_getFilterChanges` always returns an array 
+of log objects or an empty list. 
```

Comment:
Not sure if we need to mention it here, but it returns an error if the filter doesn't exist.

Reply:
We don't explicitly state this for other methods.  If we have sensible error messages, it should be clear to the user what's happened without needed to duplicate this in the doc.